king_phisher/client/dialogs/campaign_selection.py

```diff
@@ -154,7 +207,7 @@ def load_campaigns(self):
 				(hlfg_color if is_expired else fg_color),
 				(html.escape(campaign.description, quote=True) if campaign.description else None)
 			))
-		self.gobjects['label_campaign_info'].set_text("Showing {0:,} Campaign{1}".format(len(store), ('' if len(store) == 1 else 's')))
+		self.gobjects['label_campaign_info'].set_text("{0:,} campaign{1} found in the database".format(len(store), ('' if len(store) == 1 else 's')))
```

Comment:
Also if this has to be changed then it should be updated in the glade file for consistency.